src/crl.rs

```diff
@@ -262,9 +262,9 @@ impl<'a> BorrowedCertRevocationList<'a> {
                     //   values longer than 20 octets.
                     //
                     extension.value.read_all(Error::InvalidCrlNumber, |der| {
-                        let crl_number = ring::io::der::positive_integer(der)
+                        let crl_number = crate::der::nonnegative_integer(der)
```

Comment:
nit: the `crate::` prefix is an unnecessary qualification here. The `der` module is imported already.